### ISSUE TITLE
dialog show have update, but in playstore hasn't been an update there for now

### ISSUE THREAD

**Issue by eggysudianto** [timtraversy/new_version#104]
as we know, after playstore accept our app from review and publish it, the new version does not immediately appear on all devices(maybe delay)

in example, my app accept from review at 13:00, but in my phone the update is not appear yet, maybe can appear at 19:00, and in other phone can appear at 21:00, etc (random time).

can this library read update status from playstore in spesific device? not public after playstore accept our app

thanks

**Comment by timtraversy** [comment]
Probably not, but just to clarify: are you seeing the Update dialog, and then going to the Play Store and not seeing an update?

**Comment by LostInDarkMath** [comment]
@timtraversy Yes. At least for me, this is exactly the problem.

**Comment by eggysudianto** [comment]
@timtraversy Yes, in Google Play Console the review is done and my app already up to new version. but not in playstore

**Comment by blueeyestw** [comment]
> Probably not, but just to clarify: are you seeing the Update dialog, and then going to the Play Store and not seeing an update?

Yes, when I go to the play store it says "Open" instead of  "Update"

**Comment by Rodrigossff91** [comment]
This is normal, to solve this problem I put the update button later

**Comment by eggysudianto** [comment]
@Rodrigossff91 later when? how many hours after?
